### PR TITLE
Update endpoint design

### PR DESCRIPTION
**Pull request by nonoakij**
## Summary

Update endpoint design

## Reference(s)

before | after (pc) | after (mobile)
-|-|-
localhost_8000_|localhost_8000_ja_endpoints_mock%20server__selectedPageId=group-2-1-page-1 pinnedContentIds=|localhost_8000_ja_endpoints_mock%20server__selectedPageId=group-2-1-page-1 pinnedContentIds= (1)

## Checklist
- [ ] [changeset file(s)](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) included

### CODE REVIEW

File: packages/app/src/components/breadcrumb/index.tsx [hunk 1]
```diff
@@ -8,17 +8,19 @@ export type Props = BaseProps & {
...
-            <div className="">{label}</div>
-            <div className="">
-              <ChevronRightIcon className="w-em" />
-            </div>
-          </React.Fragment>
-        ))}
-      </div>
+    <div
+      className={classnames(
+        `text-thm-${on} flex items-center gap-1`,
```
Comment (nonoakij) [inline packages/app/src/components/breadcrumb/index.tsx:13]: I just fixed text color to text-thm-${on} from text-thm-on-${on}.
Other parts are refactoring, such as making the nesting shallower.

File: packages/app/src/pages/endpoints/[endpointId]/_/navigation/pages/index.tsx [hunk 2]
```diff
@@ -143,24 +135,29 @@ const Group: React.FC<{
   onSelect: (pageId: PageId) => void;
 }> = ({ pages, depth, partial, selectedPageId, onSelect }) => {
   const [isOpened, setIsOpened] = useState<boolean>(true);
-  const handleClick = useCallback<TextOnButtonProps['onClick']>(() => {
+  function handleClick() {
```
Comment (h8570rg) [inline packages/app/src/pages/endpoints/[endpointId]/_/navigation/pages/index.tsx:138]: Will we go with `function` , not arrow function ?

**Review by h8570rg — APPROVED** [review]

File: packages/app/src/pages/endpoints/[endpointId]/_/navigation/pages/index.tsx (hunk 2 above)
Comment (nonoakij) [reply packages/app/src/pages/endpoints/[endpointId]/_/navigation/pages/index.tsx:138]: I just prefer `function`. 
If you want, I will change to `arrow function`.

Comment (h8570rg) [reply packages/app/src/pages/endpoints/[endpointId]/_/navigation/pages/index.tsx]: Both will do, but i just want go with one way!

**Review by h8570rg — APPROVED** [review]

**Pull request merged**